<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fields</class>
 <widget class="QDialog" name="fields">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>22</y>
     <width>391</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>switch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>priority</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>in_port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>active</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>actions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="lineEdit_6">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>output</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>enqueue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_queue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>strip_vlan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_vlan_vid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_vlan_pcp</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>push_vlan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pop_vlan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_eth_src</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_eth_dst</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_ip_tos</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_ip_ecn</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_ipv4_src</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_ipv4_dst</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_ip_ttl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dec_ip_ttl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>copy_ip_ttl_in</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>copy_ip_ttl_out</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_mpls_label</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_mpls_tc</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_mpls_ttl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dec_mpls_ttl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>push_mpls</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pop_mpls</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>push_pbb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pop_pbb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_tp_src</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_tp_dst</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>set_field</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>meter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>copy_field</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>group</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="value">
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>fields</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>fields</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
